metabase/metabase
# Add timeseries chrome and simple date picker

Demo https://www.loom.com/share/459896063732459e8e78e44d5361ea50
Relies on https://github.com/metabase/metabase/pull/35678

How to verify:
- New -> Orders -> Sum by Count, Group by Created At (Month)
- Visualize
- You should see the timeseries chrome at the bottom (View ... by ...)
- It should be possible to apply the filter on the breakout column and change the selected temporal bucket

Notes:
- The idea is to add a "simple" variant of `DatePicker`. It uses almost the same logic, however the UI is different. I tried different approaches and the one I liked most is creating a "Simple<>" component close to the existing component and re-using its utilities and subcomponents. It makes sure these 2 will be in sync.
- I'll add tests in another PR since this is already a huge PR.

<img width="450" alt="Screenshot 2023-11-15 at 16 38 44" src="https://github.com/metabase/metabase/assets/8542534/4f689cbb-6dc4-4653-b9c3-2ccd951668a1">
<img width="691" alt="Screenshot 2023-11-15 at 16 38 50" src="https://github.com/metabase/metabase/assets/8542534/6547182a-f84f-430c-aaa2-d51c2a568238">
<img width="564" alt="Screenshot 2023-11-15 at 16 38 59" src="https://github.com/metabase/metabase/assets/8542534/bfff6ed9-99e7-4b79-80c5-d57e31c21a36">
